### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by bovee** [bovee/Aston#6]
At the minimum, we need:

- [ ] a "getting started" guide with pictures
- [ ] an advanced options guide on how to use different integrators, change peak models, etc
- [ ] with some references to where the different integrators come from, how their performance is, etc
- [ ] API documentation (for using as a Python library)

**Comment by hansyulian** [comment]
Agree... even how to open file in python is not defined here.... i have try this and try that to finally found out that this code open the file, but dont know what to do next....

from aston.tracefile import TraceFile

file = TraceFile("DAD1A.ch")
print file.ftype
